### PR TITLE
Document UID requirements and Docker troubleshooting steps.

### PR DESCRIPTION
**Pull request by markfickett**
All, please add `export UID` to your `.bashrc` for safety when dealing with your Docker volumes!

Josh and David: Please review for correctness/terminology.

Brian F: Does this align in any way with the Docker issues you were having?

Dan: FYI since you've been helping debug.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,3 +179,47 @@ Finally, write a new changelog file in `api/db/changelog/` and include it in
...
+
+If you get permissions errors like
+`Exception in thread "main" java.io.FileNotFoundException: /.gradle/wrapper/dists/gradle-4.0-all/ac27o8rbd0ic8ih41or9l32mv/gradle-4.0-all.zip.lck (Permission denied)` or from MySQL like
+`ERROR 1130 (HY000): Host '172.18.0.3' is not allowed to connect to this MySQL server`
+it may be because you ran a `docker` command as root (the default) whereas
+our `project.rb` scripts run as you. To clear permissions in Docker volumes,
+remove the volumes:
+
+```
+docker volume rm $(docker volume ls -q)
```
Comment (markfickett) [inline README.md]: Maybe change this to only `api_db` and/or workbench volumes. `docker volume rm $(docker volume ls -q | grep api_)` ?

Comment (freemabd1) [reply README.md]: Hey Mark, If i remember correctly this was the error i was getting: Exception in thread "main" java.io.FileNotFoundException: /.gradle/wrapper/dists/gradle-4.0-all/ac27o8rbd0ic8ih41or9l32mv/gradle-4.0-all.zip.lck (Permission denied). Seems that ./project docker-clean fixed my problems. I don't have very much experience with docker, so i'm not exactly sure why i was getting this error.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ Docker must be installed to build and run code (For Google workstations, see
 go/installdocker.). Ruby is required to run our development scripts, which
 document common operations and provide a convenient way to perform them.
 
+Add `export UID` to your `.bashrc`. All our Docker commands run as your user,
```
Comment (dmohs) [inline README.md]: I would prefer that we do not recommend doing this. It could hide bugs that we might otherwise fix.

**Comment by markfickett** [comment]
OK -- sounds like this may not be general enough advice to bake into our README.

**Pull request closed without merging**